index.html

```diff
@@ -12,38 +12,77 @@
     <title>Atomist | Software delivery information for cloud-native applications</title>
   </head>
   <body>
-
+      
+<!-- Collapsed Navigation -->
+      
+<div class="position-fixed" style="width: 100%;height:100vh;z-index:20;visibility: collapse" id="nav-side">
+    <img src="src/assets/atomist_logo_shapes_banner-home.jpg" style="position:absolute;z-index:-10;left:-1500px;">
```

Comment:
As a general rule, I think it's best to limit the number of `style` we use in the html. For example, you could take everything that's on the `style`, inside: `<div class="position-fixed" style="width: 100%;height:100vh;z-index:20;visibility: collapse" id="nav-side">` and put it in a css class, and then you would only need to refer to the class name in the html.